### ISSUE TITLE
Support authenticated media

### ISSUE THREAD

**Issue by divVerent** [SchildiChat/schildichat-desktop#263]
I cannot see ANY images anymore - they ALL show up as "Error downloading image". Even images I just sent myself.

I am on matrix.org homeserver and app.schildi.chat web client.

So, steps to repro: join ANY room. Post a picture. Picture immediately turns into "Error downloading image".

Is there a chance SchildiChat never implemented the requirements for MSC3916 which came into effect September 4, 2024?

https://matrix.org/blog/2024/06/26/sunsetting-unauthenticated-media/

If so, any ETA for a fix?

**Comment by divVerent** [comment]
When I just tried it out, I observed in network inspector a 404 fetch of this URL:

	https://matrix-client.matrix.org/_matrix/media/r0/download/matrix.org/myofOhgsYopLzsEuGolsYuWx

This does look like the deprecated thing.

**Comment by SpiritCroc** [comment]
Known issue, SchildiChat-desktop hasn't been updated to support [authenticated media](https://matrix.org/blog/2024/06/26/sunsetting-unauthenticated-media/) yet, which matrix.org started enforcing recently.

**Comment by SpiritCroc** [comment]
Supported in https://github.com/SchildiChat/schildichat-desktop/releases/tag/v1.11.81-sc.0.test.0 .

**Comment by SpiritCroc** [comment]
(If someone wants to patch our old codebase, we're still open for contributions, but we figured it's not worth spending time on it from our side when we really should get a more recent Element codebase running)

**Comment by steadfasterX** [comment]
this makes schildi chat unusable for me atm. it seems this also affects avatar images, e.g. coming from a (whatsapp) bridge. really hope the new codebase stabilizes soon ;)

**Comment by SpiritCroc** [comment]
> really hope the new codebase stabilizes soon ;)

Current alpha builds are as "stable" as it gets, they're separate mostly due to missing lots of our original changes, so people can install both next to each other if wanted. I wouldn't count on that changing soon

**Comment by steadfasterX** [comment]
yes I am running the latest [v1.11.86-sc.0.test.0](https://github.com/SchildiChat/schildichat-desktop/releases/tag/v1.11.86-sc.0.test.0) now which works for me (atm).
only thing which annoys me a LOT is https://github.com/element-hq/element-desktop/issues/1710 which (afaiu) will not be fixed

**Comment by sypper-pit** [comment]
image not open in linux KDE neon 6.2

**Comment by aslmx** [comment]
Whats the schedule that this is considered stable for official release?
Will the Flathub version be bumped then in a timely manner, too?

Really annyoing this issue and I'm really glad to read the bug is fixed and we just need to wait for the release, thanks a lot for this :+1:

**Comment by SpiritCroc** [comment]
There's no schedule. We still need to evaluate at what point the downsides of the outdated codebase outweigh the disadvantages of the new codebase being very slimmed down compared to previous releases.

**Comment by SpiritCroc** [comment]
Re-opening until we have new stable releases, maybe more people find it themselves then.

**Comment by fakeacc4** [comment]
Same problem here. Ty for reopen.

**Comment by MarcelCoding** [comment]
I have been successfully running https://github.com/SchildiChat/schildichat-desktop/commit/a9f81d8ebe8fec6217ab7552e5678cba25ce91ef on NixOS for two weeks. Would be interested in upstreaming the package again, once it's "released" again.

If anyone is interested, cherry-pick the two SchildiChat commits from https://github.com/NuschtOS/nuschtpkgs/commits/backports-24.11/ onto nixpkgs.

**Comment by Domoel** [comment]
Same issue for me and good to know whats behind it...

**Comment by lephisto** [comment]
ETA to fix? Schildi Desktop broken since weeks.

**Comment by SpiritCroc** [comment]
You can use the alpha builds now. For stable - no ETAs, don't expect anything soon.

**Comment by Domoel** [comment]
Maybe a cheese workaround but just use "enable_authenticated_media: false" in your homeserver.yaml for now to avoid any problems.

**Comment by denis-dordoigne** [comment]
> yes I am running the latest [v1.11.86-sc.0.test.0](https://github.com/SchildiChat/schildichat-desktop/releases/tag/v1.11.86-sc.0.test.0) now which works for me (atm). only thing which annoys me a LOT is [element-hq/element-desktop#1710](https://github.com/element-hq/element-desktop/issues/1710) which (afaiu) will not be fixed

I tried the alpha release (`schildichat-desktop-alpha_1.11.86-sc.0.test.0_amd64.deb`) and it doesn't work for me (but it works from schildichat-android or element-desktop).

**Comment by DJCrashdummy** [comment]
i also suffer from this issue which is IMHO quite a regression by itself (and justifies updating)...

> We still need to evaluate at what point the downsides of the outdated codebase outweigh the disadvantages of the new codebase being very slimmed down compared to previous releases.

but i not only wanted to whine about this issue, but also report in the sense of the mentioned quote that it seems managing sessions (especially logout) is also broken. ☹️

**Comment by SpiritCroc** [comment]
To be fair Schildi-Desktop is not really actively maintained right now, and the alpha builds don't change that much compared to Element Web anyway, one may consider switching to Element at this point. If I've heard correctly they have a unified room list now too

**Comment by Ma5t3rful** [comment]
2026 Any updates on this? I have the exact same problem. When can we expect the next stable?

```
SchildiChat version: 1.11.36-sc.3
Olm version: 3.2.14
```

**Comment by SpiritCroc** [comment]
> 2026 Any updates on this? I have the exact same problem. When can we expect the next stable?
> 
> ```
> SchildiChat version: 1.11.36-sc.3
> Olm version: 3.2.14
> ```

This project is probably going to be archived soon due to being effectively unmaintained, while current development focuses on https://github.com/SchildiChat/schildi-revenge .

**Comment by tylo** [comment]
```
SchildiChat version: 1.11.36-sc.3
Olm version: 3.2.14
```

Dang, really? I really like the UI a lot more compared to Element too. That's a bummer.
